code/modules/projectiles/gun_system.dm

```diff
@@ -844,7 +844,7 @@ and you're good to go.
 			user.apply_damage(200, STAMINA)
 		else
 			user.apply_damage(projectile_to_fire.damage * 2.5, projectile_to_fire.ammo.damage_type, "head", 0, TRUE)
-			user.apply_damage(100, OXY)
+			user.apply_damage(200, OXY)
```

Comment:
undocumented change, put on CL

Reply:
> undocumented change, put on CL

![image](https://user-images.githubusercontent.com/25491240/128420149-a83aab71-886a-40b9-b244-eb4719380313.png)
 ??

Reply:
Just edited that in a minute ago.

Reply:
ah ok haha